internal/util/idlocker.go

```diff
@@ -53,6 +53,7 @@ func (vl *VolumeLocks) TryAcquire(volumeID string) bool {
 	return true
 }
 
+// Release acquires the lock, deletes the volume and the releases the lock again
```

Comment:
nit: **then** releases the lock again

Also, go comments must end with a period

Reply:
Thanks, I'll make the change